tests/test_types.py

```diff
@@ -4,6 +4,7 @@
 from compas.geometry import Line
 from compas.geometry import Plane
 from compas.geometry import Polygon
+from compas.geometry import Polyline
```

Comment:
please be consistent with the imports, either import at top of the module or at each test function
I've slight preference to the test function level